Generator/Commands/RouteGenerator.php

```diff
@@ -90,6 +90,18 @@ public function getUserInputs()
 
         $routename = Str::lower($ui . '_' . $this->containerName . '_' . Str::snake($operation));
 
+        if ($ui == 'api') {
+            $this->stubName = 'routes/api.stub';
+        }
+
+        if ($ui == 'web') {
```

Comment:
I think this block should be refactored.. Just use the `routes/web.stub` file.. You can even "merge" both IF statements.. to just use

`$this->stubName = 'routes/' . $ui . '.stub'; `